specification/postgresql/resource-manager/Microsoft.DBforPostgreSQL/stable/2018-06-01/examples/TopQueryStatisticsGet.json

```diff
@@ -0,0 +1,32 @@
+{
+  "parameters": {
+    "api-version": "2018-06-01-privatepreview",
+    "subscriptionId": "ffffffff-ffff-ffff-ffff-ffffffffffff",
+    "resourceGroupName": "testResourceGroupName",
+    "serverName": "testServerName",
+    "queryStatisticId": "66-636923268000000000-636923277000000000-avg-duration"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "id": "/subscriptions/ffffffff-ffff-ffff-ffff-ffffffffffff/resourceGroups/testResourceGroupName/providers/Microsoft.DBforPostgreSQL/servers/testServerName/queryStatistic/66-636923268000000000-636923277000000000-avg-duration",
```

Comment:
resource type name is wrong in these examples

Reply:
The resource type being returned is a queryStatistic, topQueryStatistics is an aggregate over the querystatistics